docs/support/managing-ckeditor-logo.md

```diff
@@ -22,46 +22,46 @@ To remove the logo, you need to obtain a commercial license and then configure t
 
 Refer to the {@link support/license-key-and-activation License key and activation} guide for details on where to find the license key and how to use it in your configuration.
 
-## How to configure the layout of the "Powered by CKEditor" logo?
+## How to configure the "Powered by CKEditor" logo?
 
 For open source, free users, the "Powered by CKEditor" logo will always be displayed. There is, however, some degree of control over it.
 
-Complete configuration reference is available in the {@link module:core/editor/editorconfig~EditorConfig#ui API documentation}. In short, you can configure the following properties:
+The first area of customization is the built-in configuration. The complete reference is available in the {@link module:core/editor/editorconfig~EditorConfig#ui API documentation}. The other way of manupulating how the logo looks are CSS styles. Below you will find the examples for both directions.
```

Comment:
We can simplify this and remove paragraphs in this line (with exception of a reference to full config - which should be moved to the "Layout customization" section).